cmd/plural/version.go

```diff
@@ -7,23 +7,24 @@ import (
 	"net/http"
 	"runtime"
 	"strings"
-	"time"
 
 	"github.com/pluralsh/plural/pkg/utils"
 	"github.com/urfave/cli"
 )
 
-var (
-	GitCommit string
-	Version   string
+const (
+	devVersion = "dev"
```

Comment:
I think the logic to detect a dev version isn't fully correct. I get this after running `make install`:

```console
david@david-pop-os:~/Plural/plural-cli$ plural version
PLURAL CLI:
   version      v0.4.4-74-ge462cd9
   git commit   e462cd9
   compiled at  2022-08-09T11:41:56+0000
   os/arch      linux/amd64

Your version appears out of date, try updating it with your package manager
```

We should probably have message for dev version telling people they are using a dev version and pointing them to where they can install official releases.

Reply:
>I think the logic to detect a dev version isn't fully correct.

It shows `dev` if the build was done without any params, so without using any of the `make` commands. 

It was intended as I think in most of the cases we should avoid putting `dev` placeholder here and be as precise as possible. That's because most common use cases I see for this are checking for updates, debugging and reporting issues where every commit makes difference.

We can change that and show `dev` if the git distance is bigger than 0 but that would put all non-release versions into one bracket where only difference would be compilation time. In my opinion it would be better to avoid that and do the thing that you have suggested instead:

> We should probably have message for dev version telling people they are using a dev version and pointing them to where they can install official releases.

With this info we can warn people about using dev/non-official release but still keep all the detailed version info. What do you think?

Reply:
That sounds good to me.

Reply:
It's done:

<img width="907" alt="Zrzut ekranu 2022-08-10 o 09 16 33" src="https://user-images.githubusercontent.com/2823399/183839051-296ab3e0-8619-41b3-97d1-2ac9e8301fc5.png">